tests/unit/test_batch.py

```
@@ -0,0 +1,840 @@
+            # Should never have more than 1 other concurrent operation (total 2)
+            assert concurrent_count <= 1
+
+    def test_progress_bar_integration(self):
+        """Test that progress bar works correctly."""
+        with patch('rich.progress.Progress') as mock_progress_class:
+            mock_progress = Mock()
+            mock_task = Mock()
+            mock_progress.add_task.return_value = mock_task
+            mock_progress_class.return_value = mock_progress
+
+            def mock_operation(owner, repo, pr_number):
```

Ruff (0.13.1)

Unused function argument: `owner`

(ARG001)